app/controllers/Controller.scala

```diff
@@ -52,12 +55,65 @@ class Controller @Inject() (
     implicit request: Request[_] =>
       af.authorised() {
 
-        Future.successful(Ok("manageOrTrack"))
+        manageOrTrackView(vrn, manageOrTrackForm.fill(ManageOrTrack(None, vrn.value)))
+
+      }.recoverWith {
```

Comment:
I think we should wrap it into play Action in near time. This becomes even more urgent when we introduce this logic with enrolment checks.

 `af.authorised() { ...}.recoverWith {case e: AuthorisationException => authorisationException(e)}`

Reply:
Done